build/release.sh

```diff
@@ -15,8 +15,8 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
   # build
   VERSION=$VERSION npm run build
 
-  # TODO
-  # npm test
+  # test
+  npm run test
```

Comment:
`npm test` is special in that it is already an alias for `npm run test`, but this works!